include/vrapp/VolumeVisualizationApp.h

```diff
@@ -15,6 +15,8 @@
 #include <future>
 #include "../interaction/ArcBall.h"
```

Comment:
For consistency, want to also used the `include_directories` for these so you don't need the relative includes in this file